.drone.yml

```diff
@@ -33,9 +33,55 @@ steps:
   - mvn sonar:sonar -Dsonar.host.url=$${SONAR_HOST} -Dsonar.login=$${SONAR_TOKEN} -DprojectKeySuffix=$${projectSuffix}
   when:
     event:
-      exclude:
+      exlcude:
+      - pull_request
+
+- name: publish
+  image: maven:3.8-openjdk-17
+#  depends_on:
+#    - sonar
+  environment:
+    ARTIFACTORY_TOKEN:
+        from_secret: artifactory_token
+  commands:
+  - mvn -s ./jparest_settings.xml -f pom.xml install -B org.apache.maven.plugins:maven-deploy-plugin:2.8.2:deploy -DskipTests
+  when:
+    branch: main
+#    event:
+#      exlcude:
+#      - pull_request
+
+- name: Git Tag
+  image: jlrigau/maven-git
+  depends_on:
+    - publish
+  environment:
+    SSH_KEY:
+      from_secret: github_token
+  commands:
+    - chmod 777 git_tag.sh
+    - ./git_tag.sh
+  when:
+    branch: main
+    event:
+      exlcude:
       - pull_request
 
+- name: snapshot
+  image: maven:3.8-openjdk-17
+  depends_on:
+    - sonar
+  environment:
+    ARTIFACTORY_TOKEN:
+      from_secret: artifactory_token
+  commands:
+    - "BRANCH=$(echo $DRONE_BRANCH | tr '[:lower:]' '[:upper:]') && [[ $BRANCH =~ (EAHW-[0-9]*) ]] && snapshotSuffix=$${match/#E/-E}$${BASH_REMATCH/#E/-E}"
```

Comment:
Is there somewhere we can see an example of the snapshot name this command generates?